cyclus/system.py

```diff
@@ -12,7 +12,7 @@ def PY_VERSION_TUPLE():
 
 @lazyobject
 def curio():
```

Comment:
Yes, but if we aren't having them lazily created, the functions should be removed and the imports should happen at the module level, non-lazily.